testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/OptionsPrinter.java

```diff
@@ -112,11 +112,9 @@ else if (annotation.negatable()) {
             if (((Map<?, ?>) value).isEmpty()) {
                 return null;
             }
-            else {
-                return ((Map<?, ?>) value).keySet().stream()
-                        .map(key -> String.format("%s %s=%s", annotation.names()[0], key, ((Map<?, ?>) value).get(key)))
-                        .collect(joining(" "));
-            }
+            return ((Map<?, ?>) value).keySet().stream()
```

Comment:
remaining question is -- do we need `if (((Map<?, ?>) value).isEmpty())` at all

Reply:
Yes, removing it means that if no extra options are provided the test run commands printed by suite describe look like (the green is after removing the `isEmpty` branch):

```diff
@@ -1,8 +1,9 @@
 testing/trino-product-tests-launcher/bin/run-launcher test run --config config-default \
 --server-package core/trino-server/target/trino-server-359-SNAPSHOT.tar.gz \
 --bind \
 --output PRINT \
 --test-jar testing/trino-product-tests/target/trino-product-tests-359-SNAPSHOT-executable.jar \
 --environment multinode \
+ \
 --reports-dir presto-product-tests/target/suite-1/config-default/multinode \
 -- -x big_query,storage_formats,profile_specific_tests,tpcds,hive_compression
```

Reply:
thanks for showing this to me